### DISCUSSION TITLE
Optifabric

### DISCUSSION THREAD

**Discussion by TheProGaming** [gnembon/fabric-carpet#1782] [Q&A]
When i put in carpet mod with optifabric and optifine and launch the game i get the error 1. Everything works without the optifabric.

**Comment by hexadecimal233** [comment]
You can use sodium and other alternative mods, they have a better performance and compatibility

**Comment by altrisi** [comment]
Optifine currently breaks with Carpet, there's a fix in their side but it hasn't been released yet, it's on one of their PR.
